### ISSUE TITLE
Employee Dashboard

### ISSUE THREAD

**Issue by kelslundberg** [betheluniversity/portal#66]
Working on some sort of widget/channel that launches employees into the Employee Dashboard in SSB.

I've added some content to a channel here to get us started:
https://portal.bethel.edu/employment

But wondering if we want to make it look more prominent since it'll probably be one of the main things employees use in the portal? 

It could look similar in style to the homepage widget we'll be doing for the enrollment checklist?

Let me know your thoughts, @johjacb!

**Comment by kelslundberg** [comment]
Updates on this from our meeting yesterday. I'm moving this to high priority cause we're wanting to sit down with HR within the next couple weeks to show them where things are at...

We talked about a group of tiles/widgets next to the event feed for employees that directs people to the resources listed below.

I'm not sure @c-schwarze and @johjacb how you want to set this up - if it should be hard coded like the top of the tab currently is...I'm assuming these resources wouldn't change often.

Email 
Links to mail.bethel.edu
@c-schwarze to check on if we can pull in number of unread email

Calendar 
links to google calendar - not sure of the direct link to personal google calendar?

Drive 
Links to drive.bethel.edu

Employee Dashboard
View leave reports, time sheets, pay stubs, and other employment info.
Links to https://banner.xp.bethel.edu/EmployeeSelfService/ssb/hrDashboard#/hrDashboard%20%20

E-Announcements
Link to e-annoucements archive - I'm not sure of that link:
https://www.bethel.edu/e-announcements/archive

![screen shot 2017-06-06 at 8 05 48 am](https://cloud.githubusercontent.com/assets/6525391/26831311/8c18b040-4a91-11e7-82fe-4846bd89a239.png)

**Comment by c-schwarze** [comment]
Are we wanting icons for any/all of these?

**Comment by johjacb** [comment]
I think those can be added later, once the layout for this section is done.

**Comment by c-schwarze** [comment]
I am having @tim-heck work on this. fyi

**Comment by kelslundberg** [comment]
Updates from our meeting yesterday:

Box of 4 on the left:
Mail, Calendar, Drive, Offices

Single box on the right:
Employee Dashboard
View leave reports, time sheets, pay stubs, and other employment info.
Launch Dashboard (button? or maybe the whole thing is a button?)

**Comment by tim-heck** [comment]
![screen shot 2017-06-15 at 8 26 00 am](https://user-images.githubusercontent.com/16583548/27183492-58a9da3a-51a4-11e7-9441-785dcb47fcfb.png)
Here's what I got for the Employee Dashboard, what do you guys think?

**Comment by e-jameson** [comment]
This is looking awesome! I wonder if we want to call the right-box something else? "Employee Dashboard" and "Go to Dashboard" are kind of conflicting. Thoughts @kelslundberg @hamtim?

**Comment by kelslundberg** [comment]
I think I'll have Erik look at it from a content perspective! @hamtim - open to your thoughts as well.

I don't know if we were thinking time balances would go here or in statements and balances. We can certainly adjust anything at this point based on what we think makes good sense.

**Comment by hamtim** [comment]
The sentence, balances, and button don't seem to flow well from a content standpoint. We can chat about that next week.

**Comment by kelslundberg** [comment]
Yeah I think too a different color for the dashboard box would be good - to make it different than the other boxes to the left. Erik's looking at content...we can chat on Monday

**Comment by kelslundberg** [comment]
From our convo yesterday, it sounds like we're going to keep the 4 items - adjusting the text just a little bit:

Email
Calendar
Drive
Offices

Then, Erik is providing updated text for the employee dashboard box.

**Comment by ErikGruber** [comment]
Hey @c-schwarze, here's some content for the Employee Dashboard widget - 

Heading:
Employee Dashboard
 
Content:
View your Employee Dashboard for important employee information, including:
 
- Leave Balances
- Leave Reports
- Timesheets
- Pay Stubs
- Tax Information
- Benefits

CTA:
My Employee Dashboard

**Comment by kelslundberg** [comment]
After our lengthy discussion yesterday, we're moving a different direction with this for a few different reasons.

G-Suite and Offices are moving to tools and resources.

@johjacb has an updated layout for the Employee Dashboard section. You can use the copy Erik provided above. @ErikGruber - Maybe we want to shorten the list of what you'll find in Banner? Something like:

Leave Reports
Timesheets
Tax Information
And more

@e-jameson is connecting with Derek on data for leave balances to generate the charts

**Comment by kelslundberg** [comment]
We're meeting with HR at 8 a.m. on Thursday, July 6. 

We'll need an updated version of this ready for that meeting. It's okay if we don't have the charts working 100% as long as we can explain how they will be generated.

**Comment by kelslundberg** [comment]
@c-schwarze - can we update the events area so it's employee holidays from the calendar

The heading can also change to Upcoming Holidays

**Comment by c-schwarze** [comment]
This was updated a long time ago, but I figured I should comment.

The employee holidays events are hooked up and its renamed to Upcoming Holidays.

**Comment by kelslundberg** [comment]
I think this is ready for testing. @d-freed can you test with some employee usernames?

**Comment by johjacb** [comment]
this isn't quite ready yet, the charts are still having some JS work done
